guides/source/configuring.md

```diff
@@ -1488,6 +1488,8 @@ Define an `Array` specifying the key/value tags to be inserted in an SQL comment
 `[ :application, :controller, :action, :job ]`. The available tags are: `:application`, `:controller`,
 `:namespaced_controller`, `:action`, `:job`, and `:source_location`.
 
+WARNING: Calculating the caller location via `:source_location` is a costly operation and should be used primarily in development (note, there is also a `config.active_record.verbose_query_logs` that serves the same purpose) or occasionally on production for debugging purposes.
```

Comment:
```suggestion
WARNING: Calculating the `:source_location` of a query can be slow, so you should consider its impact if using it in a production environment.
```

This is super subjective, but I think we can be more concise (more descriptive and less prescriptive). What do you think about this?